src/SourceLink.Common/build/Microsoft.SourceLink.Common.props

```diff
@@ -7,14 +7,13 @@
 
   <PropertyGroup>
     <!--
-      Do not query source control manager for any information when building in the IDE or for Live Unit Testing.
-      Features that need this information will be disabled if false.
+      Used to suppress querying source control and features that use the information (e.g. git commit SHA).
     -->
-    <EnableSourceControlManagerQueries Condition="'$(EnableSourceControlManagerQueries)' == '' and '$(DesignTimeBuild)' != 'true' and '$(BuildingForLiveUnitTesting)' != 'true'">true</EnableSourceControlManagerQueries>
```

Comment:
This seems expensive to run at design time, so I'd want someone from project-system to agree this is OK. @kvenkatrajan can you route?

Reply:
@tmeschter please review.

Reply:
To retrieve commit sha we do read a couple of files. However, we cache repository information in msbuild cache: https://github.com/dotnet/sourcelink/blob/4ca4236c10047416251be48339d464f15a697e8b/src/Microsoft.Build.Tasks.Git/RepositoryTask.cs#L96-L105

Reply:
Actually, there is a potential for more IO. Embedding untracked sources is more expensive as it needs to iterate over all Compile items.

```
  <Target Name="_SetEmbeddedFilesFromSourceControlManagerUntrackedFiles"
          DependsOnTargets="BeforeCompile;SetEmbeddedFilesFromSourceControlManagerUntrackedFiles"
          BeforeTargets="CoreCompile"
          Condition="'$(EmbedUntrackedSources)' == 'true' and '$(EmbedAllSources)' != 'true' and '$(DebugType)' != 'none' and '$(EnableSourceControlManagerQueries)' == 'true'" />
```

Since embedding only affects PDB I think it would make sense to disable this target in DTB.